### DISCUSSION TITLE
SAWarning: Cannot correctly sort tables; there are unresolvable cycles between tables [Table A, TableB]

### DISCUSSION THREAD

**Discussion by machuwey** [agronholm/sqlacodegen#345] [Q&A]
### Things to check first

- [X] I have searched the existing issues and didn't find my feature already requested there

### Feature description

Hello, 
was wondering how could I solve this error, or specify which table should be created first.
```
SAWarning: Cannot correctly sort tables; there are unresolvable cycles between tables "A, B", which is usually caused by mutually dependent foreign key constraints.  Foreign key constraints involving these tables will not be considered; this warning may raise an error in a future release.
  for table in self.metadata.sorted_tables:

```

### Use case

Be able to generate classes from mutually referenced tables.

**Comment by agronholm** [comment]
That's a warning, not an error. Is there something wrong with the result you're getting?
